src/sass/components/_path.scss

```diff
@@ -45,25 +44,21 @@ $color-background-path-current: $brand-background-primary;
     }
   }
 
-  .slds-is-current {
-    background-color: $color-background-path-current;
+  .slds-is-active {
```

Comment:
we don't need current anymore since active is always applied when current is present.